protocol/messenger_bookmarks.go

```diff
@@ -38,3 +38,8 @@ func (m *Messenger) UpdateBookmark(ctx context.Context, oldURL string, bookmark
 	}
 	return m.SyncBookmark(ctx, &bookmark)
 }
+
+func (m *Messenger) GarbageCollectRemovedBookmarks() error {
+	threshold := uint64(time.Now().AddDate(0, 0, -14).Unix())
```

Comment:
We probably want to bump this to 30 days, same time we guarantee messages to be on mailservers

Reply:
Updated!